core/unit_test/tstSort.hpp

```diff
@@ -385,7 +385,7 @@ void testBinByMemberDataOnly()
     // Bin the aosoa by the 1D member. Use one bin per data point to
     // effectively make this a sort. Don't actually move the particle data
     // though - just create the binning data.
-    auto bin_data = Cabana::binByMember( aosoa.slice<1>(), num_data-1 );
+    auto bin_data = Cabana::binByKey( aosoa.slice<1>(), num_data-1 );
```

Comment:
Is this planned to be the last version of the `sort` interface before we lock it down?

Reply:
(not that I have a problem with it, I just hope to play around with it a bit for my PIC use case if possible before marrying down to it. As per our last discussion on this I think we had my requirements captured)

Reply:
This PR is hopefully the last update to the code API before tagging a release candidate. This captures your requirements I think but I wanted to make it more consistent in terms of naming convention.